recipes/4-Examples/extra_examples/ablation_demo.ipynb

```diff
@@ -0,0 +1,1116 @@
+       "education-num   continuous              0     1.0       20.0       True   \n",
+       "marital-status     nominal              0     NaN        NaN      False   \n",
+       "occupation         nominal              0     NaN        NaN      False   \n",
+       "relationship       nominal              0     NaN        NaN      False   \n",
+       "race               nominal              0     NaN        NaN      False   \n",
+       "sex                nominal              0     NaN        NaN      False   \n",
+       "capital-gain    continuous              0     0.0   162755.0       True   \n",
+       "capital-loss    continuous              0     0.0     8103.0       True   \n",
+       "hours-per-week  continuous              0     1.0      148.0       True   \n",
+       "native-country     nominal              0     NaN        NaN      False   \n",
+       "target             nominal              0     NaN        NaN      False   \n",
+       "\n",
+       "               data_type original_type       \n",
+       "                             data_type size  \n",
+       "age               number       numeric    8  \n",
+       "workclass         number       integer    8  \n",
+       "fnlwgt            number       numeric    8  \n",
+       "education         number       integer    8  \n",
+       "education-num     number       numeric    8  \n",
+       "marital-status    number       integer    8  \n",
+       "occupation        number       integer    8  \n",
+       "relationship      number       integer    8  \n",
+       "race              number       integer    8  \n",
+       "sex               number       integer    8  \n",
+       "capital-gain      number       numeric    8  \n",
+       "capital-loss      number       numeric    8  \n",
+       "hours-per-week    number       numeric    8  \n",
+       "native-country    number       integer    8  \n",
+       "target            number       integer    8  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df = fetch_data(\"adult\", local_cache_dir=\"../../../data\")\n",
+    "features = infer_feature_attributes(df)\n",
+    "\n",
+    "display(df)\n",
+    "display(features.to_dataframe())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Create Trainee and Set Parameters\n",
+    "\n",
+    "The process of creating the `Trainee` is the same as the other recipes. However, after the `Trainee` is created, we set auto-analyze and auto-ablation parameters to ensure that the `Trainee` remains informed as it removes cases during train."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Version 30.1.0 of Howso Engine™ is available. You are using version 30.0.9.dev9+g369bbb6.d20241016.\n",
```

Comment:
Re-run necessary to get rid of this warning?

Reply:
I was gonna merge and do a re-run on everything